_episodes/03-intro-git-github.md

```diff
@@ -13,6 +13,194 @@ keypoints: ""
 
 ## Basic Workflow
 
+TODO:
+
+- [x] list diagram images
+- [x] intersperse `git` commands corresponding to each diagram part
+- [ ] make diagram for `git revert`
+- [ ] make example for `git revert`
+- [ ] walk through to check flow of diagrams with `git` commands
+- [ ] add example code (including outputs) to commands
+- [ ] if time allows / needed, adjust diagram image dimensions
+
+Diagrams to go with the basic commands.
+
+(starting with only a working directory, coordinate with R plotting contents)
+
+NOTE (from RD): this image only applicable when the starting working directory is currently empty
+
+![git-basics-flow-01-opt1](../fig/git-basics/20201117-git-01-opt1.png)  
+
+```
+git init
+{: .language-bash}
+```
```

Comment:
Wherever there's a code chunk, we'll need the language specifier after the closing backticks instead of before.